src/common/datastore.js

```
@@ -385,38 +385,37 @@ var DataStore = function() {
   },
 
   /**
-   * Get the Pbk of the WA.
-   * @ return the pbk.
+   * Get the Certificate of the WA.
+   * @ return the public certificate.
    */
-  this.getPbkApplication = function(appToken, callback) {
+  this.getCertificateApplication = function(appToken, callback) {
     var appToken = appToken.toString();
-    log.debug('datastore::getPbkApplication --> Going to find the pbk for the appToken ' + appToken);
+    log.debug('datastore::getCertificateApplication --> Going to find the certificate for the appToken ' + appToken);
     this.db.collection('apps', function(err, collection) {
       callback = helpers.checkCallback(callback);
       if (err) {
-        log.error('datastore::getPbkApplication --> there was a problem opening the apps collection: ' + err);
+        log.error('datastore::getCertificateApplication --> there was a problem opening the apps collection: ' + err);
         callback(err);
         return;
       }
       collection.findOne({ _id: appToken }, function(err, data) {
         if (err) {
-          log.error('datastore::getPbkApplication --> There was a problem finding the PbK - ' + err);
+          log.error('datastore::getCertificateApplication --> There was a problem finding the certificate - ' + err);
           callback(err);
           return;
         }
         if (!data) {
           log.debug('There are no appToken=' + appToken + ' in the DDBB');
-          callback(null, null);
+          callback('There are no appToken=' + appToken + ' in the DDBB', null);
```

we shall return an error cause if not, server brokes when this error happens.